packages/react-i18n/babel.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/babel-plugin');
```

Comment:
You'd need to add this + the .d.ts file to the `files` array in `package.sjon`

Reply:
this package doesn't have a files array. should I add one?